src/SDK/Common/Util/DanglingSpansShutdownHandler.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\SDK\Common\Util;
+
+use \OpenTelemetry\Context\DebugScope;
+use OpenTelemetry\SDK\Trace\Context;
+
+/**
+ * SDK helper classes for dealing with dangling spans
+ */
+final class DanglingSpansShutdownHandler
+{
+    /**
+     * Function to remove any contexts that may have been left over by early exits where
+     * scope->detach was not called. e.g. An early exit() call from an PHP script/request.
+     *
+     * To register this call:
+     * ShutdownHandler::register(DanglingSpansShutdownHandler::shutdown)
+     *
+     * BUT make sure you do this before you register the other shutdown handlers or any spans won't be processed/exported
+     *
+     * @return void
+     */
+    public static function shutdown(callable $fn): void
```

Comment:
it doesn't look like `$fn` is used now, can it be removed?

Reply:
yes, I thought I had. will fix.